pkg/api/core/v1/client/apps.go

```diff
@@ -276,6 +276,24 @@ func (c *Client) AppUpload(namespace string, name string, tarball string) (model
 	return resp, nil
 }
 
+// AppValidateCV uploads vlaidates the chart values of the specified app against its appchart
```

Comment:
```suggestion
// AppValidateCV uploads validates the chart values of the specified app against its appchart
```

Reply:
uploads or validates? strange syntax there

Reply:
Stronger typo. The `uploads` should and will be removed.

Reply:
Done 082bb31a3d9357a6b384c682472c6e3cf85f253c